### PR TITLE
Feat add suggestion comp

### PR DESCRIPTION
**Pull request by RubenMateus**

### CODE REVIEW

**Comment by clucasalcantara** [comment]
Also you gonna need a rebase since there's changes already in master

File: src/components/molecules/suggestion/Suggestion.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const Suggestion = ({ title, subTitle, buttonText }) => (
+	<section className="wwc-hightlight">
+		<div className="ui container center aligned">
+			<div className="ui placeholder segment">
+				<div className="ui icon header">
+					<i aria-hidden="true" className="coffee icon"></i>{title}
+					<div className="sub header">{subTitle}</div>
+				</div>
+				<div className="inline">
+					<Button text={buttonText}/>
```
[the remark this answers is not in the document]
Comment (RubenMateus) [reply src/components/molecules/suggestion/Suggestion.js]: makes sense will do that.
Related to the layout, there is an issue, the footer is sticky and is over the last element on the page, thats why i added the card after the suggestion, according to the general layout #1

[the remark this answers is not in the document]
Comment (RubenMateus) [reply src/components/molecules/suggestion/Suggestion.js]: And also, the deafult prop for handleclick should what?

[the remark this answers is not in the document]
Comment (RubenMateus) [reply src/components/molecules/suggestion/Suggestion.js]: @clucasalcantara should we use `;`?

[the remark this answers is not in the document]
Comment (clucasalcantara) [reply src/components/molecules/suggestion/Suggestion.js]: You could but the lint/prettier should remove it :P

[the remark this answers is not in the document]
Comment (clucasalcantara) [reply src/components/molecules/suggestion/Suggestion.js]: The default prop for the function could be a simple () => {}

**Pull request merged**